apps/web/src/modules/auction/components/CurrentAuction/PlaceBid.tsx

```diff
@@ -182,16 +183,38 @@ export const PlaceBid = ({
               </Box>
             </Box>
           </form>
-
-          <ContractButton
-            className={auctionActionButtonVariants['bid']}
-            handleClick={handleCreateBid}
-            disabled={address && isValidChain ? !isValidBid : false}
-            mt={{ '@initial': 'x2', '@768': 'x0' }}
-          >
-            Place bid
-          </ContractButton>
-        </Fragment>
+          <Flex mt="x2">
+            <ContractButton
+              className={auctionActionButtonVariants['bid']}
+              handleClick={handleCreateBid}
+              disabled={address && isValidChain ? !isValidBid : false}
+              mt={{ '@initial': 'x2', '@768': 'x0' }}
+            >
+              Place bid
+            </ContractButton>
+            {chain.id !== 1 ? (
+              <Button
```

Comment:
We need a new `auctionActionButtonVariants` entry for this button. it needs to be styled as a secondary button